docker/graalvm/Dockerfile

```diff
@@ -1,5 +1,6 @@
 
 FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-21.0.0
+ARG VERSION="latest"
```

Comment:
would "dev" be a better version?  If `VERSION` isn't set it's likely because it was hand-build outside of the build scripts, which sets `VERSION'

Reply:
`dev` sounds better, using that.